app/controllers/workflow_executions/submissions_controller.rb

```diff
@@ -3,11 +3,14 @@
 module WorkflowExecutions
   # Workflow submission controller
   class SubmissionsController < ApplicationController
+    include Irida::Pipelines
+    cattr_reader :available_pipelines
```

Comment:
**nitpick (blocking):** I think you accidentally added this line e.g. `cattr_reader :available_pipelines` as there is no available_pipelines here and instead it is defined in `Irida::Pipelines`

Reply:
Yup definitely was added accidentally. Updated to call Irida::Pipelines.* directly and removed the include and cattr_accessor in d08fd2d05586261bfb12aafc47e7e971a61f711f